opm/utility/ECLCaseUtilities.cpp

```diff
@@ -0,0 +1,291 @@
+#include <opm/utility/ECLCaseUtilities.hpp>
+
+#include <exception>
+#include <initializer_list>
+#include <iomanip>
+#include <regex>
+#include <sstream>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+#include <boost/filesystem.hpp>
+
+#include <ert/ecl/ecl_file.h>
+#include <ert/ecl/ecl_file_kw.h>
+#include <ert/ecl/ecl_file_view.h>
+#include <ert/util/ert_unique_ptr.hpp>
+
+class IsSeparateRestart
+{
+public:
+    IsSeparateRestart(const std::string& prefix)
+        : restart_(prefix + R"~~(\.(F|X)\d{4}$)~~")
```

Comment:
Please add human-readable comment about what the regexp is doing

Reply:
:+1:

Reply:
> Please add human-readable comment about what the regexp is doing

Updated in commit 9fc70e0.